src/views/statutory/QuestionLinesList.vue

```diff
@@ -12,9 +12,6 @@
 
         <b-table :data="questionLines" :loading="isLoading" :selected.sync="selectedQuestionLine" narrowed>
           <template slot-scope="props">
-            <b-table-column field="id" label="#" numeric>
-              {{ props.row.id }}
```

Comment:
can you instead replace it with `props.index + 1`? that should work

Reply:
That's also a possibility, but I'm not sure why we would need visible IDs to the question lines

Reply:
I remember Chair asking for it, but don't remember why